.github/workflows/canary.yml

```diff
@@ -0,0 +1,99 @@
+name: Publish canary on PR comment
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  check_auth:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/canary' }}
```

Comment:
Thoughts on `/canary` to request a canary build or alternatives? We could potentially add a way to trigger like we do with the other workflows, or maybe even observe some label on the PR (that one is a bit more far-fetched).